### ISSUE TITLE
Components / Lists - Element IDs are not unique

### ISSUE THREAD

**Issue by yesenia-gtz** [Office-of-Digital-Services/California-State-Web-Template-Website#975]
[Components / Lists
](https://as-cdt-pub-migr-betaws-ww-p-001-stage.azurewebsites.net/components/lists.html)

Two or more elements on this page have the same id value. These should be unique.

![Screen Shot 2022-12-13 at 1 22 33 PM](https://user-images.githubusercontent.com/98193284/207446285-2baee114-96e7-4702-868f-18cf01daf0ec.png)

- [ ] Make one of the ID elements unique. 

#tabs1
#tab-content
#tabs2
#tab-content-2

**Comment by extinah** [comment]
Assigning to @carterm as lead. Please assign to a dev, as needed. Send to Pia for review/more info.

**Comment by kkoryaka** [comment]
@carterm I will take care of it

**Comment by kkoryaka** [comment]
@pglomboy this is now updated in staging: https://as-cdt-pub-migr-templatews-ww-p-001-dev.azurewebsites.net/components/lists.html

**Comment by pglomboy** [comment]
@kkoryaka looks good and I see the elements id are unique to each other
